src/Altinn.App.Core/Features/Payment/Processors/Nets/NetsMapper.cs

```diff
@@ -62,6 +62,54 @@ internal static class NetsMapper
         };
     }
 
+    /// <summary>
+    /// Map from out Payer type to NetsCheckoutConsumerDetails.
+    /// </summary>
+    public static NetsCheckoutConsumerDetails? MapConsumerDetails(Payer? payer)
+    {
+        if (payer == null)
```

Comment:
I think in general we've started moving away from `==` and `!=` in favor of `is` and `is not`. Just some food for thought :)